README.rst

```diff
@@ -1,4 +1,4 @@
-# PyLmod
+PyLmod
 ========
 :PyLmod: Python implementation of MIT Learning Modules API
 :Version: 0.1.0
```

Comment:
We should pull this.  The badge will have this automagically from pypi once we release